.hound.yml

```diff
@@ -5,4 +5,6 @@ rubocop:
 
 reek:
   enabled: true
-  config_file: .reek
+
+  config_file: .reek.yml
```

Comment:
Do not change this file!

Reply:
I needed to change those files so now they are the same as updated in summer-2019